modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BuilderScreen/BuilderScreen.tsx

```diff
@@ -151,9 +155,11 @@ export function BuilderScreen({
 												  )
 											: filter
 											? viewColumn?.objectFieldBusinessType
-											: viewColumn?.label[
-													defaultLanguageId
-											  ]
+											: getLocalizableLabel(
+													creationLanguageId!,
```

Comment:
@alinedoleron Could you replace `creationLanguageId!` with `creationLanguageId as Locale`? It's good to avoid using the ! because it can obfuscate problems with undefined variables. Also, please check other places you're using it in this PR. Thanks :)